src/solution-workbooks/tkgm-to-tkgs-migration-reference-design.md

```diff
@@ -0,0 +1,273 @@
+# Reference Design for Migration from TKGm to TKGs (vSphere with Tanzu)
+
+Currently over 120+ customers are currently utilising VMware Tanzu Multi-Cloud Grid TKGm. With the proposed roadmap on TKGm continuity, existing customers need to have a mechanism to migrate to vSphere with Tanzu to retain the business continuity. The mechanism should enable seamless migration of Core Applications, utilizing their existing Infrastructure, Storage, Network, Security Policies, RBAC, authentication policies. The mechanism should also address very minimal or no additional purchase of software and solutions but utilize the existing solutions offerings from VMware.
```

Comment:
```suggestion
With the proposed roadmap on Tanzu Kubernetes Grid with a management cluster (known informally as TKGm) continuity, existing customers need to have a mechanism to migrate to vSphere with Tanzu to retain the business continuity. The mechanism should enable seamless migration of Core Applications, utilizing their existing Infrastructure, Storage, Network, Security Policies, RBAC, authentication policies. The mechanism should also address very minimal or no additional purchase of software and solutions but utilize the existing solutions offerings from VMware.
```

Reply:
> 

Thanks for the feedback Stuart, we have updated this in the doc.